daffodil-cli/src/it/scala/org/apache/daffodil/tunables/TestCLITunables2.scala

```diff
@@ -80,4 +82,52 @@ class TestCLITunables2 {
       shell.close()
     }
   }
+
+  @Test def test_CLI_Parsing_ReloadingDoesNotRepeatWarnings(): Unit = {
+    val schemaFile = Util.daffodilPath("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/suppressWarnTest.dfdl.xsd")
+    val testSchemaFile = if (Util.isWindows) Util.cmdConvert(schemaFile) else schemaFile
+    val compiledProcFile = Util.newTempFile("savedProc", ".bin")
+    compiledProcFile.deleteOnExit()
+    val compiledProcFilePath = compiledProcFile.getAbsolutePath()
+    //
+    do {
+      val shell = Util.start("")
+      try {
+        // note: 2>&1 is shell-speak for "connect stderr into stdout"
+        val cmd = String.format("""%s save-parser -s %s %s 2>&1""", Util.binPath, testSchemaFile, compiledProcFilePath)
```

Comment:
Expect keeps track of the stderr and stdout separately, so you don't have to do this redirect. The `expect()` function accepts a an index for which stream to expect from. It defaults to 0 (i.e. stdout) if not specified, but you can speicfy 1 for stderr. So if you remove the redirect, you can change the below expect to be:

```scala
shell.expect(1, contains("""Schema Definition Warning"""))
```

Reply:
I tried this.

`shell.expect(1,....)`

Very problematic. There are two overloads which take an integer. The first is deprecated and takes a long. That one the long is a timeout. The other takes an int. That one the int is an output stream number. 

I even tried

`shell.expect(1.toInt,...)`

but they still indicate I'm calling the deprecated one where the first arg is a timeout. 

Until they actually remove the deprecated one, 
I think I will stick with 2>&1.

Reply:
Ah, I had the wrong function name. It's `expectIn`, we use it in a handful of places so it should work.